src/CodeStyle/CSharp/Tests/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "xUnit.net Console": {
+      "executablePath": "$(XUnitPath)",
+      "commandLineArgs": "$(XUnitArguments)"
+    }
+  }
+}
```

Comment:
I'd rather not check in 50 copies of this. Is there a reason linked files didn't work?

If likned files don't work can we get a bug on project system trakcing this problem. It's a *huge* step backwards.